sys/freebsd/pf.txt

```diff
@@ -143,3 +152,9 @@ pfioc_states_v2 {
 	ps_req_version	int64
 	psu_buf		ptr[out, array[int8]]
 }
+
+pfioc_nv {
+	data		ptr[in, array[int8]]
+	len		int64
+	size		int64
```

Comment:
The `len` type can be used here to make the description more precise. In particular, I think this could be:

```
pfioc_nv {
    data ptr[in, array[int8]]
    len int64
    size len[data]
}
```